### PR TITLE
Mo check out test

### PR DESCRIPTION
**Pull request by Moelnur**
-Created basket.feature page
- Created BasketStepDefs for testing quantity in the basket
- Created BasketPage for methods on basket page
- Created TShirtPage for methods on the selected item

### CODE REVIEW

**Review by tuhelahmed98 — APPROVED** [review]
Fantastic work:)

File: src/test/java/stepDefinitions/BasketStepDefs.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        basketPage = new BasketPage();
+        tShirtPage = new TShirtPage();
+    }
+
+    @Given("I am on the homepage")
+    public void iAmOnHomePage(){
+        homePage.goToHomepage();
+    }
+    @When("I select an item")
+public void clickOnItem(){
```
Comment (matthewmccormick) [inline src/test/java/stepDefinitions/BasketStepDefs.java:26]: @Moelnur Can you reformat this file please before merging

**Review by matthewmccormick — CHANGES_REQUESTED** [review]

**Review by selena92 — APPROVED** [review]
Nice work :)

**Comment by SamRouse123** [comment]
Great work!

**Review by matthewmccormick — APPROVED** [review]

**Pull request merged**